streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceivedBlockTracker.scala

```diff
@@ -40,7 +42,8 @@ private[streaming] case class BatchAllocationEvent(time: Time, allocatedBlocks:
   extends ReceivedBlockTrackerLogEvent
 private[streaming] case class BatchCleanupEvent(times: Seq[Time])
   extends ReceivedBlockTrackerLogEvent
-
+private[streaming] case class CombinedRBTLogEvent(events: List[ReceivedBlockTrackerLogEvent])
```

Comment:
Could you use `Seq` rather than `List`? Then you don't need to convert an `ArrayBuffer` to `List`.